### DISCUSSION TITLE
Cypress logs show localhost:<randomport> instead of real URL

### DISCUSSION THREAD

**Discussion by sagi-rika** [cypress-io/cypress#16929] [General]
Hi,

I have been using Cypress for a while now and I'm really satisfied. I use env vars to configure my baseUrl, and its working great. Lately, I've noticed that the logs of cy.visit (for example), show that the domain of the request is, for example, `localhost:62312` instead of `something.sandbox.net`.

Is that on purpose? I wish to see the real url for debugging purposes.

I should note that the same fake domain is shown in the console's address bar while the tests run.

thanks!

**Comment by jennifer-shehane** [comment]
I'd need a reproducible example to exactly understand, but it sounds like the `baseUrl` is not being correctly set if the main windows url is localhost + random port. https://on.cypress.io/best-practices#Without-baseUrl-set-Cypress-loads-main-window-in-localhost-random-port

**Reply by sagi-rika** [reply]
You ar ecorrect. I was setting `baseUrl` inside the env property, instead of the top level
